src/frontend/src/components/SelectLanguageButtons.tsx

```diff
@@ -0,0 +1,40 @@
+import { useEffect, useState } from "react";
+
+import {
+    RadioButton,
+    RadioButtonGroup,
+} from "./base/RadioButtonGroup/RadioButtonGroup";
+
+export const Languages = {
+    EN: "en",
+    ES: "es",
+    ZH: "zh",
+};
+
+type LanguageKeys = (typeof Languages)[keyof typeof Languages];
+
+interface Props {
+    language: LanguageKeys;
+}
+
+const SelectLanguageButtons = ({ language }: Props) => {
+    const [selectedLanguage, setSelectedLanguage] =
+        useState<LanguageKeys>(language);
+
+    useEffect(() => {
+        setSelectedLanguage(language);
+    }, [language]);
+
+    return (
+        <RadioButtonGroup
+            value={selectedLanguage}
+            onChange={setSelectedLanguage}
+        >
+            <RadioButton value={Languages.EN} label="English" />
```

Comment:
Since all we're doing in our implementation of `RadioButton` is initialize styles, we should otherwise try to match the API of the underlying React Aria component. [They pass the label as a child](https://react-spectrum.adobe.com/react-aria/RadioGroup.html#value), not as a prop. Consider updating `RadioButton` to match that:

```suggestion
            <RadioButton value={Languages.EN}>English</RadioButton>
```

Reply:
Agreed. Addressed in https://github.com/azavea/echo-locator/pull/672/commits/380fbe03ce2f249c0a5da92355258e6f6a902c3d